### DISCUSSION TITLE
Can not Create a Next.js app

### DISCUSSION THREAD

**Discussion by LucasColas** [vercel/next.js#54669] [Help]
### Summary

Hello,
I'm trying to learn next js and react js.
I'm following this [documentation](https://nextjs.org/learn/basics/create-nextjs-app/setup).
However I can't run that line : 
`npx create-next-app@latest nextjs-blog --use-npm --example "https://github.com/vercel/next-learn/tree/master/basics/learn-starter"`

So I went [there](https://github.com/vercel/next-learn/blob/master/basics/errors/install.md).
I also tried this command : 
`npx create-next-app nextjs-blog`

I also checked this [documentation](https://nextjs.org/docs/getting-started/installation).
And I tried to run : 
`npx create-next-app@latest`

The three command lines give the same error : 
```
npm ERR! code ENOENT
npm ERR! syscall lstat
npm ERR! path C:\Users\lucas\AppData\Roaming\npm
npm ERR! errno -4058
npm ERR! enoent ENOENT: no such file or directory, lstat 'C:\Users\lucas\AppData\Roaming\npm'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent
```
Of course I installed node js. 
If I write : 
`npm`
it works.

My node js verison is 20.5.1.
I'm using windows 11.

**Comment by Nadeera3784** [comment]
Hi @LucasColas,
have you tried to clear the npm cache?
`npm cache clean --force`
